tests/sles4sap/hana_test.pm

```diff
@@ -46,6 +46,12 @@ sub run {
 
     assert_script_run "HDB info";
 
+    # Check for bsc#1031355
+    # UserTasksMax should be set to "infinity" in /etc/systemd/logind.conf.d/sap.conf
+    my $uid = script_output = "id -u";
```

Comment:
`script_output = "id-u"`, are you sure? :)

Reply:
As @ldevulder says, syntax for `script_output()` is wrong.

Reply:
Fixed.